pvlib/data/adr-library.csv

```diff
@@ -0,0 +1,1762 @@
+Name,Manufacturer,Model,Source,Vac,Vintage,Pacmax,Pnom,Vnom,Vmin,Vmax,ADRCoefficients,Pnt,Vdcmax,Idcmax,MPPTLow,MPPTHi,TambLow,TambHi,Weight
+0,0,0,0,0.0,0.0,0.0,0,0,0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
```

Comment:
Yea exactly. I could write a conditional statement in ```_parse_raw_sam_df``` to only skip rows for certain csvs, but maybe you can think of something better than that?